docs/sections/calculations/index.rst

```diff
@@ -40,57 +38,49 @@ Calculation processes
     :link-badge:`https://aiida-vasp.readthedocs.io/en/latest/calculations/vasp.html, "VASP",cls=badge-vasp text-white, tooltip=Go to the VASP version.`
 
 .. panels::
-    :header: header-text
+    :header: panel-header-text
     :body: bg-light
     :footer: bg-light border-0
 
     ------
-    :column: col-lg-12
+    :column: col-lg-6
 
     .. link-button:: errors
         :type: ref
-        :text: Calculation Errors
+        :text: Troubleshooting errors
         :classes: btn-light text-left stretched-link font-weight-bold
     ^^^^^^^^^^^^
 
-    *"It's the job that's never started as takes longest to finish."* |br|
-    *-- Samwise Gamgee*
-
-    Calculations can fail! Learn how to deal with this.
+    This module will take you trough some of the common errors that can happen when submitting calculations and how to deal with them.
 
     +++++++++++++
     .. list-table::
         :widths: 50 50
         :class: footer-table
         :header-rows: 0
 
-        * - |time| 20 min
-          - Intermediate
-
-.. panels::
-    :header: header-text
-    :body: bg-light
-    :footer: bg-light border-0
+        * - |time| 15 min
+          - |aiida| :aiida-blue:`Intermediate`
 
     ------
-    :column: col-lg-12
+    :column: col-lg-6
 
     .. link-button:: workflows
         :type: ref
-        :text: Running your first workflow
+        :text: Running workflows
         :classes: btn-light text-left stretched-link font-weight-bold
     ^^^^^^^^^^^^
 
-    Start with running your first workflow.
+    Once you have know how to submit calculations, this module will teach you how to extend that to workflows.
```

Comment:
```suggestion
    Once you know how to submit calculations, you can learn how to launch workflows in this module.
```

Reply:
I seem to have some difficulties verbing